### PR TITLE
Add some informational help text

### LINKED ISSUE #32: Add homepage content
Currently the homepage just displays a version string.  Add buttons to get to sections of the app accessible in the hamburger menu or something else more visually appealing.

### PR DESCRIPTION
**Pull request by sjenks**
Fixes #32 
So people can RTFM
This fixes no bugs ;)

### CODE REVIEW

File: src/Views/Home.css [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-  text-decoration: none;
-}
-
-.Home-content a:hover {
-  color: #888;
+  color: #0000ff;
+  text-decoration: underline;
 }
 
 .Home-content ul {
```
Comment (kfarnung) [inline src/Views/Home.css]: You can probably use default styling if it looks OK. This was for the first navigation before bootstrap.

File: src/Views/Home.tsx [hunk 2]
```diff
@@ -1,26 +1,70 @@
...
+          <h2>
+            What type of puzzle are you trying to solve?
+          </h2>
+
+          <h3>
+            Ciphers - Scrambled Letters
+          </h3>
+          <ul>
+            <li>
+              <a href="/#/cipher/caesar">Caesar Cipher</a> - Letters are shifted by a fixed value
```
Comment (kfarnung) [inline src/Views/Home.tsx]: I think react-router has a link element that's preferred.

File: src/Views/Home.tsx [hunk 3]
```diff
@@ -1,26 +1,70 @@
...
-              Puzzle Hunt
-            </a>, <a href="http://www.mit.edu/%7Epuzzle/" target="_blank">
-              Mystery Hunt
-            </a>, <a href="http://www.puzzledpint.com/" target="_blank">
-              Puzzled Pint
-            </a>, and many others.
-          </p>
-        </div>
+          <h2>
+            What type of puzzle are you trying to solve?
```
Comment (kfarnung) [inline src/Views/Home.tsx]: Feels like it's time to be more data driven, maybe build a hierarchy of pages and categories with metadata and use it for generating both navigation views?

**Comment by beckbria** [comment]
> So people can RTFM
> This fixes no bugs ;)

Wrong, this seems to do #32   :P

File: src/Views/Home.tsx (hunk 3 above)
Comment (beckbria) [reply src/Views/Home.tsx]: Agreed - if nothing else, having a common source for the strings to come from makes sense.  Who knows, if we get bored at some point in the future, we might even try to add localization......

File: src/Data/RouteData.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    public static getCategories() {
+        return RouteData.getRoutes().categories;
+    }
+
+    private static getRoutes() {
+        const routes = {
+            'name': 'Puzztool',
+            'categories': [
+                {
+                    'name': 'Ciphers - Scrambled Letters',
```
Comment (kfarnung) [inline src/Data/RouteData.ts]: Can you make these match the menu option names and move the description to a separate field? Once this land in planning to use the same data to generate the menus as well.

Comment (kfarnung) [reply src/Data/RouteData.ts]: Also for completeness can you add the root url (E.g. '/cipher'), it's used for menu highlighting.

File: src/Data/RouteData.ts [hunk 5]
```diff
@@ -8,77 +8,83 @@ export class RouteData {
...
-                            'url': '/cipher/autokey',
+                            'url': '/autokey',
                             'description': ' A Vignere cipher which uses plaintext as the key'
                         }
                     ]
                 },
                 {
-                    'name': 'Encoding - Different Ways To Represent Letters',
+                    'name': 'Encodings',
+                    'description': 'Different Ways To Represent Letters',
```
Comment (beckbria) [inline src/Data/RouteData.ts]: Casing is inconsistent - The description of Ciphers only capitalizes the first word, while this does a lot more.  I prefer Sentence casing (only capitalizing the first one), but please standardize on one.

File: src/Data/RouteData.ts [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+                            'name': 'Pigpen',
+                            'url': '/pigpen',
+                            'description': 'Parts of a tic-tac-toe grid or X, some with dot'
+                        }
+        
+                    ]
+                },
+                {
+                    'name': 'Reference',
+                    'description': 'Misc information',
```
Comment (beckbria) [inline src/Data/RouteData.ts]: Please spell out Miscellaneous.

File: src/Views/Cipher/Autokey.tsx [hunk 7]
```diff
@@ -7,6 +7,14 @@ class Autokey extends React.Component {
     return (
       <div className="Autokey">
         <div className="Autokey-content">
+          <h2>Autokey Cipher</h2>
+          <p>
+            An autokey cipher is similar to a Vigenere cipher, where the plaintext is used after the key runs out. 
+            This way the repetition of a Vigenere cipher is solved.
```
Comment (beckbria) [inline src/Views/Cipher/Autokey.tsx]: Nit: "is solved" is a bit ambiguous here.  Consider "This addresses the key length repetition issues with the Vigenere cipher."

File: src/Views/Cipher/Autokey.tsx [hunk 8]
```diff
@@ -7,6 +7,14 @@ class Autokey extends React.Component {
...
+          </p>
+          <p>
+            Enter the text and key, and select encrypt or decrypt
```
Comment (beckbria) [inline src/Views/Cipher/Autokey.tsx]: Missing period.

File: src/Views/Cipher/Caesar.tsx [hunk 9]
```diff
@@ -7,6 +7,14 @@ class Caesar extends React.Component {
     return (
       <div className="Caesar">
         <div className="Caesar-content">
+          <h2>Caesar Cipher</h2>
+          <p>
+            A Caesar cipher works by shifting every letter by a fixed amount.  
+            This is a simple cipher - there are only 25 different shifts for letter.
```
Comment (beckbria) [inline src/Views/Cipher/Caesar.tsx]: "for letters" or "for each letter".

File: src/Views/Encoding/AutoConvert.tsx [hunk 10]
```diff
@@ -7,6 +7,21 @@ class AutoConvert extends React.Component {
     return (
       <div className="AutoConvert">
         <div className="AutoConvert-content">
+          <h2>AutoConvert</h2>
+          <p>
+            AutoConvert will automatically detect and convert several encodings - simply enter your ciphertext.
+            If all of your text is in the same encoding, you can select "force consistent encoding".  
+            This can be useful for ambigious strings
```
Comment (beckbria) [inline src/Views/Encoding/AutoConvert.tsx]: Missing period.

**Pull request merged**